user-service/README.md

```diff
@@ -0,0 +1,77 @@
+# User Service
+
+The user service stores aggregate information on a per user basis. For each user we will record the number of replays, wins, losses, goals, shots, saves, and more. Using fast api the user and the frontend can interact with this information through various CRUD operations. Note: only the read operations are done by anyone, other actions will require authentication.
+
+## Structure
+
+This folder is broken up into:
+
+- `../config`: contains environment variables, database, and sql files
+- `../routers`: contains the routers of the service's api
+- `../schemas`: contains the schemas for your data
+- `../services`: contains the business logic of your service
+- `../tests`: contains both the unit tests and integration tests
+- `../util`: contains helper classes for handling responses 
+
+## Environment Variables
+
+To run this project, you will need to add the following environment variables to your `/user-service/.env` file and `/.env` file
```

Comment:
This is just an FYI but if a user were to run this service via docker, it will automatically create a .env from the root.